graph.py

```diff
@@ -89,12 +88,17 @@ class RawRepoData:
     _cache = TTLCache(100, ttl=_ttl)
     _last_expiry = time.monotonic()
 
-    def __init__(self, channel: str, arch: str = "linux-64", ttl=600):
+    def __init__(self, url: str, channel: str, arch: str = "linux-64", ttl=600):
```

Comment:
`url` (or whatever it ends up being called), should be a keyword argument with `"https://conda.anaconda.org/"` as the default.